.github/workflows/CI.yml

```diff
@@ -723,13 +748,50 @@ jobs:
           cd testing-framework/terraform
           make checkCacheHits
 
+  release-candidate:
+    runs-on: ubuntu-latest
```

Comment:
I think this should be removed right?

Reply:
We could also consider adding an additional parameter to the `if` statement so that we only run this job on `main` and `release/` branches. This may make the updating of dev branch easier. We would have to do this on main branch first.

Reply:
https://github.com/aws-observability/aws-otel-collector/pull/2184/files